### DISCUSSION TITLE
Make Press traps more meaner in Resonant difficulty by dropping after they get attacked by the player below outside sensor range

### DISCUSSION THREAD

**Discussion by Fluff8836** [DoukutsuRandamu/IssueTracker#83] [Suggestions]
Long-ranged weapons can hit Press traps when attacking from below, from a big distance and that is smart way to do. But, let's just make them meaner. Make players think twice when attacking from far below; if any of their bullets hit the Press in Resonant difficulty, make them attack back by dropping immediately. That will force them to get away and wait until the Press goes back up to it's resting spot. Rinse and repeat.

**Comment by TrashboxBobylev** [comment]
Satisfied by 1.44.1 (by putting falling from damage behavior for Reinforced Presses).
